lib/cms/modules.ts

```diff
@@ -179,7 +179,7 @@ export async function createModule(
     other_assets: [],
     smart_type: 'NOT_SMART',
     tags: [],
-    is_available_for_new_content: false,
+    is_available_for_new_content: moduleDefinition.availableForNewContent,
```

Comment:
For the sake of keeping the same pattern - I would destructure the `availableForNewContent` property from the module definition using the const above already doing so for a [few other things on this line](https://github.com/HubSpot/hubspot-local-dev-lib/pull/202/files#diff-34db6be7e79d62e16dc949471534ee89bdbd4aed6f75d3a7f1deea18d48f2473R164-R169)